### ISSUE TITLE
tokei panic while set with output param

### ISSUE THREAD

**Issue by oif** [Aaronepower/tokei#83]
```
rustc 1.13.0 (2c6933acc 2016-11-07)
binary: rustc
commit-hash: 2c6933acc05c61e041be764cb1331f6281993f3f
commit-date: 2016-11-07
host: x86_64-apple-darwin
release: 1.13.0
```

```
NeoMacBook-Pro:~ neo$ RUST_BACKTRACE=1 tokei Development --output json
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Syntax(KeyMustBeAString, 0, 0)', ../src/libcore/result.rs:799
stack backtrace:
   1:        0x105efa0a8 - std::sys::backtrace::tracing::imp::write::h6f1d53a70916b90d
   2:        0x105efd2ff - std::panicking::default_hook::{{closure}}::h137e876f7d3b5850
   3:        0x105efc395 - std::panicking::default_hook::h0ac3811ec7cee78c
   4:        0x105efc9a6 - std::panicking::rust_panic_with_hook::hc303199e04562edf
   5:        0x105efc844 - std::panicking::begin_panic::h6ed03353807cf54d
   6:        0x105efc762 - std::panicking::begin_panic_fmt::hc321cece241bb2f5
   7:        0x105efc6c7 - rust_begin_unwind
   8:        0x105f287b0 - core::panicking::panic_fmt::h27224b181f9f037f
   9:        0x105d71b1a - core::result::unwrap_failed::h9916702f61d59849
  10:        0x105da3f30 - tokei::main::h788afbac7a995aa4
  11:        0x105efd8ba - __rust_maybe_catch_panic
  12:        0x105efbed6 - std::rt::lang_start::h538f8960e7644c80
```

And so sorry for the inconvenience about posted on  wrong place.

**Comment by Aaronepower** [comment]
See this serde_json pull request, once that is merged and released, this will be fixed. https://github.com/serde-rs/json/pull/172

**Comment by dtolnay** [comment]
I released serde_json 0.8.4 containing a fix.
